boards/weact-f401ce/doc.txt

```diff
@@ -0,0 +1,70 @@
+/**
+@defgroup    boards_weact-f401ce WeAct-F401CE board
+@ingroup     boards
+@brief       Support for the WeAct-F401CE Board
+
+## Overview
+
+WeAct-F401CE is a board with the same form-factor as the blue/blackpill,
+but with an STM32F401CEU6 and a USB-C connector.
+
+It is available on sites like AliExpress for less than 3€.
```

Comment:
Is this information important ? What about non EU members ?

Reply:
I copied that from `blxxxpill` and adapted the price - can drop it.

Reply:
I'm not insisting, it's just that RIOT users are not necessarily in EU :)

Reply:
AliExpress ships worldwide. And now that even James Bond villians prefer EUR over USD (see movie Quantum of Solace (2008)), we can expect everyone being able to make use of that info :-)

Reply:
🇪🇺🇪🇺🇪🇺